Lib9c/ActionObsoleteConfig.cs

```diff
@@ -64,11 +64,10 @@ public static class ActionObsoleteConfig
 
         public const long V100310ExecutedBlockIndex = 5_217_577L;
 
-        // Current: 6_974_550 at 2023-06-14 16:10:00 KST
-        // Release plan: About 7_102_350 at 2023-06-29 12:00:00 KST
-        // Seconds per block: 10
         public const long V200030ObsoleteIndex = 7_200_000;
 
+        public const long V200031ObsoleteIndex = 7_205_000;
```

Comment:
It should bump this index. Now, tip index is 7204613.

Reply:
Thanks a lot!

Reply:
I've fix it to 7,206,000.